### PR TITLE
feat(admin-ui): make admin-ui compatible with latest version node and npm

### PR DESCRIPTION
**Pull request by jv18creator**
issue: #899 
issue: #952

### CODE REVIEW

File: admin-ui/app/components/Layout/Layout.js [hunk 1]
```diff
@@ -239,7 +239,7 @@ class Layout extends React.Component {
           changeMeta: (metaData) => { this.setState(metaData) }
         }}
       >
-        <Helmet>
+        {/* <Helmet>
```
Comment (mjatin-dev) [inline admin-ui/app/components/Layout/Layout.js:242]: Please remove commented code.

File: admin-ui/app/styles/miltonbo/scss/_animations.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                        box-shadow 200ms ease-in-out;
+        }
+    }
+
+    .sidbear-menu .badge {
+        transition: opacity 200ms ease-in-out;
+    }
+
+    /*
+    &.sidebar--slim.sidebar--collapsed {
```
Comment (mjatin-dev) [inline admin-ui/app/styles/miltonbo/scss/_animations.scss:47]: Please remove commented code.

File: admin-ui/plugins/admin/components/CustomScripts/CustomScriptForm.js [hunk 3]
```diff
@@ -23,6 +23,7 @@ import { useTranslation } from 'react-i18next'
 import items from './scriptTypes'
 
 function CustomScriptForm({ item, scripts, handleSubmit, viewOnly }) {
+  console.log(scripts,"here")
```
Comment (mjatin-dev) [inline admin-ui/plugins/admin/components/CustomScripts/CustomScriptForm.js:29]: Please remove console.log

File: admin-ui/app/components/BlockUi/BlockUi.js [hunk 4]
```diff
@@ -0,0 +1,215 @@
+import React, { Component } from 'react';
```
[the remark this answers is not in the document]
Comment (jv18creator) [reply admin-ui/app/components/BlockUi/BlockUi.js]: alright, I've made some changes and converted the lib component in Functional component. could you please review the changes?

**Pull request merged**